concepts/stringers/about.md

```diff
@@ -0,0 +1,68 @@
+# About
+
+[Stringer][stringer-interface] is an interface for defining the string format of values.
+
+The interface consists of a single `String` method:
+ 
+```go
+type Stringer interface {
+    String() string
+}
+```
+ 
+The [fmt][fmt-package] package (and many others) will look for this interface to print values.
+
+## Example: Distances
+
+Assume we are working on an application that deals with geographical distances measured in different units. 
+We have defined types `DistanceUnit` and `Distance` as follows: 
+ 
+```go 
+type DistanceUnit string
+
+const (
+	Kilometer    DistanceUnit = 0
+	Mile         DistanceUnit = 1
+	NauticalMile DistanceUnit = 2
```

Comment:
This might be a silly question but how come we could assign a `DistanceUnit` type (mask over `string`) to an integer? According to [the enum link](https://yourbasic.org/golang/iota/#complete-enum-type-with-strings-best-practice) you provided below, this `DistanceUnit` should still be an `int` type?

Reply:
Not a silly question at all - it is a bug  - a leftover from an earlier version.
I have corrected this to `type DistanceUnit int`.